### PR TITLE
Add comments feature

### PR DESCRIPTION
**Pull request by szymonjanota**

### CODE REVIEW

File: src/routes/lotteryComments.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import express from 'express';
+import { pick } from 'lodash';
+
+import Lottery from '../models/Lottery';
+import Comment from '../models/Comment';
```
Comment (lukix) [inline src/routes/lotteryComments.js]: Please, export Comment model from `models/index.js` and import here using:
`import { Lottery, Comment } from '../models';`

File: src/routes/lotteryComments.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+const createComment = async (req, res, next) => {
+  console.log(req.session);
+  try {
+    const { lotteryId } = req.params;
+    const { id: authorId } = req.session.user;
+    const {
+      author,
+      content,
+    } = pick(req.body, ADD_COMMENT_PERMITTED_PARAMS);
+    console.log(lotteryId, authorId, author, content);
```
Comment (lukix) [inline src/routes/lotteryComments.js]: Remove console.log

File: src/routes/lotteryComments.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+const ADD_COMMENT_PERMITTED_PARAMS = [
+  'lottery',
+  'author',
+  'content',
+];
+
+const getComments = async (req, res, next) => {
+  try {
+    const { id: userId } = req.session.user;
+    console.log(req.params, req.query);
```
Comment (lukix) [inline src/routes/lotteryComments.js]: Remove console.log

File: config/index.js [hunk 4]
```diff
@@ -4,4 +4,5 @@ module.exports = {
   PORT: process.env.PORT || 3000,
   DB_URL: process.env.DB_URL || 'mongodb://localhost:27017/randomixerdb',
   MONGOOSE_RESTART_TIMEOUT: process.env.MONGOOSE_RESTART_TIMEOUT || 5000,
+  SESSION_SECRET: process.env.SESSION_SECRET || 'SESSION_SECRET',
```
Comment (marcinlesek) [inline config/index.js]: I see you add global variable, please also add it in `.env`s files in `.docker` directory.

File: src/routes/lotteryComments.js (hunk 2 above)
Comment (marcinlesek) [reply src/routes/lotteryComments.js]: @szymonjanota remove this console.log also please.

File: src/index.js [hunk 5]
```diff
@@ -35,12 +37,33 @@ db.once('open', () => {
...
   const socket = io(server);
+  app.sockets = {
+    root: socket,
+    comments: socket.of('/comments'),
+  }
 
   initializeLotteryScheduler(lotteryId => socket.emit(`finish-${lotteryId}`));
 
+  app.use(cors({
+    origin: 'http://localhost:3000',
```
Comment (lukix) [inline src/index.js]: Move to env variable pls

**Pull request merged**